### ISSUE TITLE
It would be beneficial to reduce the number of files uploaded to S3 to reduce the Minio Metadata overhead

### ISSUE THREAD

**Issue by pidiaquez** [scylladb/scylla-manager#4167]
- new feature request

It's has been reported by users that Minio suffers from contention , reduced performance
when uploading too many sstable files,   this has been linked to the Minio Metadata updates/reads triggered, even when listing files. 

Think of backup of tables having too many sstables.

Minio does offer a feature to upload files as .zip , it will index them later and allow for individual retrival if needed

https://blog.min.io/small-file-archives/

Just sharing this for possible feature improvement

In summary the ask  is to reduce the number of files uploaded to S3, the use of the feature above is just provided as reference

**Comment by mykaul** [comment]
They should get an enterprise grade S3 service then. Seriously.  https://blocksandfiles.com/2020/09/22/ceph-scales-to-10-billion-objects/ 

zip is bad for multiple reasons. One is that our content is already compressed and presently encrypted, so zip won't add much but overhead.

I'm closing this for the time being.
